cppcache/src/SerializationRegistry.hpp

```diff
@@ -79,10 +80,10 @@ class TheTypeMap {
   std::unordered_map<internal::DSFid, TypeFactoryMethod>
       m_dataSerializableFixedIdMap;
   std::unordered_map<std::string, TypeFactoryMethodPdx> m_pdxSerializableMap;
-  mutable util::concurrent::spinlock_mutex m_dataSerializablePrimitiveMapLock;
-  mutable util::concurrent::spinlock_mutex m_dataSerializableMapLock;
-  mutable util::concurrent::spinlock_mutex m_dataSerializableFixedIdMapLock;
-  mutable util::concurrent::spinlock_mutex m_pdxSerializableMapLock;
+  mutable std::mutex m_dataSerializablePrimitiveMapLock;
```

Comment:
Given that you are touching 90% of the member variables in this class perhaps now is a good time to make them style guide compliant and more correctly named, like `dataSerializableMapMutex_`.

Reply:
I made the changes according to your proposal.
Nevertheles according to the CONTRIBUTING.md file we should be following Google's C++ style guide. According to it, the member variables should be named like ```data_serializable_map_mutex_```.
Is there any other style guide I am not aware of?

Reply:
Haha. It looks like this guide changed after we adopted it. Camel case was acceptable at the time. We should probably address this. So I guess do worry about the variables right now. Thanks for spotting that.

Reply:
Oh I see you made them anyway. Cool!